setup.py

```diff
@@ -22,7 +22,7 @@
     "ruff ~= 0.2.1",
 ]
 extras["docs"] = []
-extras["test_prod"] = ["pytest", "pytest-xdist", "pytest-subtests", "parameterized"]
+extras["test_prod"] = ["pytest>=7.2.0,<8.0.0", "pytest-xdist", "pytest-subtests", "parameterized"]
```

Comment:
```suggestion
extras["test_prod"] = ["pytest>=7.2.0,<=8.0.0", "pytest-xdist", "pytest-subtests", "parameterized"]
```

`8.0.1` is the one to avoid